uber/models/__init__.py

```diff
@@ -1546,10 +1546,6 @@ def insert_test_admin_account(self):
 
             self.add(all_access_group)
             self.add(test_developer_account)
-            self.add(AdminAccessGroup(
-                admin_account_id=test_developer_account.id,
-                access_group_id=all_access_group.id,
-            ))
```

Comment:
``AdminAccessGroup`` no longer exists, so I removed this block.  With that being said, we might need to actually replace this with something to give the same level of access.

Reply:
This was (theoretically) fixed in https://github.com/magfest/ubersystem/pull/3716/, which I'll merge now. I'll merge this after, then try to deploy and go from there.